From 614d917c313e2d8f6142c5cb1cc438ec27aa5bb5 Mon Sep 17 00:00:00 2001
From: Tony Espy <espy@canonical.com>
Date: Fri, 5 Jun 2020 14:42:18 -0400
Subject: [PATCH] Makefile: rm golangci-lint tool

Remopve golangci-lint from EXTERNAL_TOOLS as it breaks the build.
---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2721b938c..cf7a4413d 100644
--- a/Makefile
+++ b/Makefile
@@ -16,8 +16,7 @@ EXTERNAL_TOOLS_CI=\
 	golang.org/x/tools/cmd/goimports
 EXTERNAL_TOOLS=\
 	golang.org/x/tools/cmd/goimports \
-	github.com/client9/misspell/cmd/misspell \
-	github.com/golangci/golangci-lint/cmd/golangci-lint
+	github.com/client9/misspell/cmd/misspell
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v pb.go | grep -v vendor)
 
 GO_VERSION_MIN=1.12.7
-- 
2.17.1

